### ISSUE TITLE
YAxis shows no numbers when Series is wrapped in a div

### ISSUE THREAD

**Issue by mindjuice** [whawker/react-jsx-highcharts#49]
If you create a hierarchy that has a two `YAxis` objects, each with a `Series` as a child, but the `Series` objects are actually wrapped in a `<div>`, then the range of the second `YAxis` is not shown.

```jsx
          <YAxis id="price">
            <YAxis.Title>Price</YAxis.Title>
            <AreaSplineSeries id="profit" name="Profit" data={data1} />
          </YAxis>

          <YAxis id="social" opposite>
            <YAxis.Title>Social Buzz</YAxis.Title>
            <div>
              <SplineSeries id="twitter" name="Twitter mentions" data={data2} />
            </div>
          </YAxis>
```

Live Example:
https://codesandbox.io/s/p3xrwv36m0

Notice that the Social Buzz axis has no numbers.

Remove the `<div>` around the the `Series` and it works.

In my case, this is happening because I am returning one or more `Series` objects from an intermediate component that is a child of the `YAxis`, and since React 15 can't return an array, I wrapped the `Series` objects in a `<div>`.

I suspect that this won't be a problem if I upgrade to React 16, but that will require  bit of work in my app.

Is there any way to make this work with the intermediate `<div>`?

**Comment by mindjuice** [comment]
Note that this does not happen if you have a single YAxis, and wrap it in a `div`.

**Comment by whawker** [comment]
It's quite a brittle solution (I'll keep investigating), but if you can estimate the maximum individual value of the series your second axis will contain you could add a `max` prop to enforce a range to display.

```jsx
<YAxis id="social" opposite max={150}>
  <div>
    <Series id="series1" />
  </div>
</YAxis>
```

**Comment by mindjuice** [comment]
FYI, I just tried updating to React 16 and returning a Fragment instead of a `<div>`, but that doesn't work either.

Adding the `max={150}` had no effect for me.

It also doesn't matter if the second axis is `opposite` or not.  The numbers don't show either way.

**Comment by whawker** [comment]
Interesting, thanks.

This seems to work, in [this example](https://stackblitz.com/edit/react-mggihj?file=SeriesWrapper.js) I've used an intermediary component called `SeriesWrapper` that passes the `dimension` and `axisId` props through to the series. These props are usually passed from the axis to the child components (expected to be Series components).

P.s. I prefer StackBlitz to CodeSandbox, as it seems to load dependencies faster.

**Comment by mindjuice** [comment]
Thanks, the little hack of passing the `dimension` and `axisId` through did the trick! 👍 

Do you think long term these should be passed in the context instead?

**Comment by whawker** [comment]
This has been fixed as of #74 

Please `npm install --save react-jsx-highcharts@^3.0.0` to allow this behaviour
